test/connect.js

```diff
@@ -230,7 +230,8 @@ describe('connect', function() {
 			store.subscribe.args[0][0]();
 
 			component.once('stateSynced', function(data) {
-				assert.ok(!data.changes.storeState);
```

Comment:
I was kind of relying on this assert to check that the rerender wasn't happening, which wasn't really right as you pointed out. We should change this to be more reliable though, so this test can actually make sure that the child's render function is being called at all.

This could be done by using sinon.spy on the child's `render` function before calling `subscribe` and checking here that it wasn't called. Could you do this change?

I think we can remove the `notDeepEqual` call below too, since that's not the purpose of this test.